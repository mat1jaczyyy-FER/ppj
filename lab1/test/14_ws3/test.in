  
   

    

    123 3	 abc312 	abc 	123 +x 	-x


		
