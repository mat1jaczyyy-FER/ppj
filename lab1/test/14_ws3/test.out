BROJ 6 123
BROJ 6 3
IDN 6 abc312
IDN 6 abc
BROJ 6 123
OP_PLUS 6 +
IDN 6 x
OP_MINUS 6 -
IDN 6 x
